### ISSUE TITLE
Evernote Web Clipper not compatible with uBlock

### ISSUE THREAD

**Issue by MacProGirl** [uBlockOrigin/uAssets#153]
### URL(s) where the issue occurs

All sites

### Describe the issue

The Evernote Web Clipper in Firefox is broken when uBlock is enabled. Tried to uninstall it and then the web clipper worked.

I contacted Evernote and they confirmed the issue but had no idea how to solve it. I guess the uBlock crowd is very small at Evernote.

### Versions

Browser/version: Firefox 48
uBlock Origin version: 1.9.6

### Settings

Don't know how to post these.

**Comment by gorhill** [comment]
_"All sites"_? I need a URL, and details about what exactly is broken, keep in mind I know nothing about "Evernote Web Clipper".

**Comment by Hrxn** [comment]
I assume you meant this?

Chrome Extension:
https://chrome.google.com/webstore/detail/evernote-web-clipper/pioclpoplcdbaefihamjohnefbikjilc?hl=en

Firefox Extension:
https://addons.mozilla.org/en-US/firefox/addon/evernote-web-clipper/

**Comment by gwarser** [comment]
https://github.com/gorhill/uBlock/issues/2009

**Comment by gorhill** [comment]
Given that you stated [here](gorhill/uBlock#2009) that you have a "lot of 3rd party filters" and "some dynamic rules", what does [the logger](https://github.com/gorhill/uBlock/wiki/The-logger) says when the issue manifest?

**Comment by MacProGirl** [comment]
Sorry for the late answer. I've been away and missed this email. So to
answer all:

Yes it is the referred to Evernote Clipper that can be found on both
Chrome, Safari och Firefox. It is used to save all text/photos on a site or
just selected parts. So it could be used on most sites, but are not working
on any at the moment.

So I followed Raymond Hill's good advice to check the logger. But when
pushing that button nothing happens. No log window shows upp. So another
unexpected error. Any solution to this?

Thanks for this community! It is new to me and is very much appreciated!


Given that you stated here <http://gorhill/uBlock#2009> that you have a

**Comment by MacProGirl** [comment]
Did my previous email get read? How could I even found out?

I sent this question through email. I don't have the permission to post
through the website.

**Comment by IDKwhattoputhere** [comment]
> How could I even found out?

Visit GitHub. There's a link in the e-mail.

**Comment by MacProGirl** [comment]
Great! Now I see. And thanks for answering!

I reinstalled it. Didn't work. So I chose to set it back to factory
defaults. Still nothing happens when I push the logger.

It seems every tech gadget I have is doomed to failure. :(


On Mon, Oct 10, 2016 at 2:58 AM, IDKwhattoputhere



*How could I even found out?*Visit GitHub. There's a link in the e-mail.

Since the logger doesn't open have you tried reinstalling uBO?

**Comment by gorhill** [comment]
> I reinstalled it. Didn't work. So I chose to set it back to factory defaults. Still nothing happens when I push the logger.

That the logger would not open is weird. Try to delete uBO's data file, then restart the browser. See [this page](https://github.com/gorhill/uBlock/tree/master/dist#firefox) to find out the location of uBO's data file.

**Comment by MacProGirl** [comment]
I weren't able to find info regarding the uBO's data file on the page you
linked. I tried to search for the file "ublock0.sqlite" in my Mac's Finder
but couldn't find that file name either.

On Mon, Oct 10, 2016 at 4:23 AM, Raymond Hill

I reinstalled it. Didn't work. So I chose to set it back to factory

**Comment by MacProGirl** [comment]
Reaching out for help once again regarding how to delete the uBO data file.
There is no info where I can find and delete it on a Mac / OS X.

**Comment by gorhill** [comment]
Somebody with OSX will have to provide the answer. Maybe ask on some Firefox forums about where to find user profile, from there I will be able to tell you where to find uBO's SQLite file.

**Comment by MacProGirl** [comment]
I did some quick googling to find the answer, deleted the sqlite file, but
there was no difference. I then accidentally double clicked the logger,
maybe in frustration, and that was the secret sauce.

I am attaching the error message if that is to any help.


​
And when I clicked that one, the following message shows.


​I guess it says that the static filter used is found in Fanboy's Annoyance
List.

So is there any tweaking I can do to easily resolve this?

Thanks!


On Tue, Oct 18, 2016 at 1:38 PM, Raymond Hill [Masked] <
FWD-737QTYSMENAIYCQANIQDDZAD2ACIAMNIAOAAZ6ZEGBTSAJRAE2ERN3I2MQAYCACDASQBYISYOE2PGTIBCLNTSUDJKFIYACPDENZRJCCRKRWCQCRTNPYVJBSOXMDJ4PKJBSIXUEDLQRQYIIBAFIZAUIA5EA42AJNABOQALIBLVI25IAIWADLAA7A2EMFEQU3FAQQAAIA=@opayq.com

**Comment by MacProGirl** [comment]
It seems I couldn't attach photos, so here are links instead.

Here's the line from the logger:
http://imgur.com/E95Nknp

And when you click that line you get the following info:
http://imgur.com/pd0JkK4

**Comment by MacProGirl** [comment]
Also note that I have, since long before, put "evernote.com" in the uBlocks
white list.


It seems I couldn't attach photos, so here are links instead.

**Comment by MacProGirl** [comment]
Sorry to spamming the thread, but I found the solution. To put "evernote.com"
in the white list did nothing as I wrote before. But now instead I went to
the evernote.com website and I allowed that site globally in the dynamic
filters by marking it green.

Case solved! Thanks for hanging in there with me!


Also note that I have, since long before, put "evernote.com" in the uBlocks

**Comment by gorhill** [comment]
It appears you are filtering behind-the-scene network requests. This should not happen with default settings:
- Is the entry `behind-the-scene` present in the _Whitelist_ pane in the dashboard?
- Is _"Advanced user mode" enabled or disabled in the _Settings_ pane in the dashboard?

**Comment by MacProGirl** [comment]
That entry is not present in the Whitelist pane. And yes I am using
advanced mode.



On Wed, Oct 19, 2016 at 6:22 AM, Raymond Hill [Masked] <

**Comment by gorhill** [comment]
> That entry is not present in the Whitelist pane.

Why did you remove it? That is your issue. Add `behind-the-scene` in the _Whitelist_. I do warn in the wiki about messing up with advanced settings, they need to be left alone unless you understand completely what are teh consequence of changing them.

**Comment by IDKwhattoputhere** [comment]
> http://imgur.com/pd0JkK4

It's a filter from `Fanboy’s Annoyance List‎`.

**Comment by gorhill** [comment]
Duh me, that's right, I missed that link.

**Comment by MacProGirl** [comment]
I added back the default whitelist listings. I must have seen something
there that didn't sound familar and erased it maybe thinking some malware
has put the entry there. Very amateurish of me to not google before
changing something in a privacy/security suite.

Well, all seems good now so all advice was helpful and I've learned my
lessons.

Big thanks to the community and the helpful people!


On Wed, Oct 19, 2016 at 1:31 PM, Raymond Hill
